README.rst

```diff
@@ -53,28 +50,6 @@ You can verify Recaptcha input by testing the return value of::
   context.restrictedTraverse('@@captcha').verify()
 
 
-Differences between this package's API and collective.captcha
```

Comment:
Maybe leaving a note that this package was once compatible with the other would be a good thing, but this is not a real blocker.